app/assets/stylesheets/_settings.scss

```diff
@@ -57,9 +57,9 @@ $foundation-palette: (
   alert: #cc4b37,
 );
 $light-gray: #F9FAFA;
-$medium-gray: #838D91;
-$dark-gray: #3E4E54;
-$black: #0a0a0a;
+$medium-gray: darken($light-gray, 5%);
+$dark-gray: #838D91;
```

Comment:
haha, yea I know right. Not something I really like either but the variable is part of the framework. It took me a while to get over color.